README_py.md

```
@@ -1,16 +1,15 @@
 ## Table of Contents
-- [Why?](#why?)
+- [Why?](#why)
 - [How it works](#how-it-works)
-- [Requirements](#Requirements)
-  - [Creating python-env with pybind11](#Creating-python-env-with-pybind11)
-    - [Using venv (recommended)](#Using-venv)
-    - [Using anaconda](#Using-anaconda)
+- [Requirements](#requirements)
+  - [Setting up Python virtual environment](#setting-up-python-virtual-environment)
+- [Adapting this to your C++ code](#adapting-this-to-your-c-code)
 - [Examples](#Examples)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #examples; Actual: #Examples

(MD051, link-fragments)